src/smart_contracts/contract_vm.ml

```diff
@@ -261,7 +310,9 @@ module Make (CTX : Context.CTX) : S with module Context = CTX = struct
       | 7 -> load ~ctx mem x GET_CONTRACT_OPT
       | 8 -> load ~ctx mem x TRANSACTION
       | 9 -> load ~ctx mem x MINT_TICKET
-      | _ -> failwith "unimplemented"
+      | x ->
+        Format.printf "code  %d\n" x;
```

Comment:
Is that intentional? You could use `Log.debug` for that.

Reply:
need to remove them